src/main/java/org/italiangrid/storm/webdav/tpc/TransferFilterSupport.java

```diff
@@ -82,6 +76,18 @@ protected Multimap<String, String> getTransferHeaders(HttpServletRequest request
           continue;
         }
         xferHeaders.put(xferHeaderName.trim(), request.getHeader(headerName));
+      } else if (headerName.trim().equalsIgnoreCase(TransferConstants.REPR_DIGEST_HEADER)) {
```

Comment:
You could add a static import to avoid writing TransferConstants class and improve readability